config.json.sample

```diff
@@ -0,0 +1,27 @@
+{
+  "prod" : {
+    "workers": 3,
+    "port": 9124,
+    "upstream": "ws://localhost:8080",
+    "retryDelay": 10000,
+    "closeTimer": 15000,
+    "alertPath": "/somepath",
+    "alertHost": "127.0.0.1",
+    "alertPort": 9555,
+    "alertReceivers": ["test"]
+  },
```

Comment:
prod paths we can include in config.json in a different way. we can move dummy values to `test/dev`

Reply:
As discussed this will be picked from current config dir as of now. Since, this is open source later we can do cosmetics if required.